packages/fxa-event-broker/src/queueworker/queueworker.service.ts

```diff
@@ -89,14 +94,12 @@ export class QueueworkerService
       // Verify that the queue exists
       const queueParts = this.queueName.split('/');
       const queueName = queueParts[queueParts.length - 1];
-      const queues = await this.sqs.listQueues().promise();
+      const command = new ListQueuesCommand({});
```

Comment:
This returns paginated results -- any reason to expect we have > 1000 queues and need to handle this?

Reply:
I wouldn't think so, but maybe useful to add as a comment.